pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1209,33 +1209,7 @@ func isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool {
 	//In ppc64le usb devices like mouse / keyboard are set by default,
 	//so we can't disable the controller otherwise we run into the following error:
 	//"unsupported configuration: USB is disabled for this domain, but USB devices are present in the domain XML"
```

Comment:
I think this comment should be moved under the specific method of `IsUSBNeeded` for ppc64le struct

Reply:
Makes sense, i moved it.